lib/fluent/plugin/out_kafka.rb

```diff
@@ -228,11 +228,15 @@ def emit(tag, es, chain)
         partition = (@exclude_partition ? record.delete('partition'.freeze) : record['partition'.freeze]) || @default_partition
         message_key = (@exclude_message_key ? record.delete('message_key') : record['message_key']) || @default_message_key
 
-        value = @formatter_proc.call(tag, time, record)
-
-        log.trace { "message will send to #{topic} with partition_key: #{partition_key}, partition: #{partition}, message_key: #{message_key} and value: #{value}." }
+        record_buf = @formatter_proc.call(tag, time, record)
+        record_buf_bytes = record_buf.bytesize
+        if @max_send_limit_bytes && record_buf_bytes > @max_send_limit_bytes
+		  log.warn "record size exceeds max_send_limit_bytes. Skip event:", :time => time, :record => record
```

Comment:
[nits] Could you use space to indent to match other codes?

Reply:
@ganmacs Resolved!